server/src/main/java/org/spine3/server/storage/AggregateStorage.java

```diff
@@ -129,29 +130,30 @@ public void writeEvent(I id, Event event) {
         checkNotNull(event);
 
         final AggregateStorageRecord record = toStorageRecord(event);
-        writeInternal(id, record);
+        writeRecord(id, record);
     }
 
     /**
      * Writes a {@code snapshot} by an {@code aggregateId} to the storage.
      *
      * @param aggregateId an ID of an aggregate of which the snapshot is made
-     * @param snapshot the snapshot of the aggregate
+     * @param snapshot    the snapshot of the aggregate
      * @throws IllegalStateException if the storage is closed
      */
     public void write(I aggregateId, Snapshot snapshot) {
         checkNotClosed();
         checkNotNull(aggregateId);
         checkNotNull(snapshot);
 
-        final AggregateStorageRecord record = AggregateStorageRecord.newBuilder()
-                .setTimestamp(checkIsPositive(snapshot.getTimestamp(), "Snapshot timestamp"))
-                .setEventType(SNAPSHOT_TYPE_NAME)
-                .setEventId("") // No event ID for snapshots because it's not a domain event.
-                .setVersion(snapshot.getVersion())
-                .setSnapshot(snapshot)
-                .build();
-        writeInternal(aggregateId, record);
+        final AggregateStorageRecord record =
+                AggregateStorageRecord.newBuilder()
+                                      .setTimestamp(checkIsPositive(snapshot.getTimestamp(), "Snapshot timestamp"))
```

Comment:
Also consider renaming `checkIsPositive()` to `checkPositive()` to follow `checkNotNull()` and `checkNotClosed()`.